src/data/nat/prime.lean

```diff
@@ -112,6 +112,14 @@ succ_pred_eq_of_pos pp.pos
 theorem dvd_prime {p m : ℕ} (pp : prime p) : m ∣ p ↔ m = 1 ∨ m = p :=
 ⟨λ d, pp.2 m d, λ h, h.elim (λ e, e.symm ▸ one_dvd _) (λ e, e.symm ▸ dvd_refl _)⟩
 
+theorem primes_not_dvd {p q : ℕ} (pp : p.prime) (qp : q.prime) (neq : p ≠ q) : ¬(p ∣ q) :=
```

Comment:
This could be an iff (and without the `not`s).
`p = q \iff p | q`
I think it should be just as useful.